backend/main.py

```diff
@@ -13,6 +13,15 @@
 
 app = FastAPI(debug=True)
 
+origins = ["http://localhost:5000"]
```

Comment:
can you move it to config file or something so that we can change it easily while deployment